webapp/channels/src/components/multiselect/multiselect.tsx

```diff
@@ -571,3 +573,5 @@ const styles = {
         };
     },
 };
+
+export default injectIntl(MultiSelect as React.ComponentType<Props<Value> & WrappedComponentProps>);
```

Comment:
There seems to be problem because injectIntl doesnt take it a generic. Heres what i propose to change

Pass the `intl` prop to this component where it is called instead of injecting this component. 
So eg in `add_groups_to_channel_modal.tsx` where Multiselect is used

```
<MultiSelect
                        key='addGroupsToChannelKey'
                        options={groupsToShowValues}
                        optionRenderer={this.renderOpti
                        intl={this.props.intl}
                     ```
                     
                     let me know if this makes sense

Reply:
let me know if this makes sense